### PR TITLE
clarification needed

### PR DESCRIPTION
**Pull request by rdiazel**
it took me a while to find out that this does not work in 2012R2. is not supported and CMDLETS are not available

### CODE REVIEW

File: WindowsServerDocs/identity/ad-fs/operations/Configure-AD-FS-to-authenticate-users-stored-in-LDAP-directories.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ ms.prod: windows-server
 
 ms.technology: identity-adfs
 ---
-# Configure AD FS to authenticate users stored in LDAP directories
+# Configure AD FS to authenticate users stored in LDAP directories in Windows Server 2016,2019.(2012R2 is not supported)
```
Comment (illfated) [inline WindowsServerDocs/identity/ad-fs/operations/Configure-AD-FS-to-authenticate-users-stored-in-LDAP-directories.md]: just a simple suggestion of style & readability (spacing) improvement:
```suggestion
# Configure AD FS to authenticate users stored in LDAP directories in Windows Server 2016 and 2019 (2012 R2 is not supported)
```
Note that this is a header text (only 1 # in front), so a period character is not expected.

**Comment by illfated** [comment]
Looking back at the whole document, it seems that this added text may fit better as an "Applies to ..." line beneath the title line.
I will wait for feedback from the document owner or manager for a confirmation about this.

**Comment by eross-msft** [comment]
@billmath  Please review this text and let me know if it's ready to merge. Thanks!

**Comment by illfated** [comment]
@rdiazel : I have updated my suggestion, in case you want to commit that change directly. You are of course free to choose which method you want to use to change the page title.

**Comment by billmath** [comment]
@rdiazel Looks good, thanks!

#sign-off

**Pull request merged**